easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-intel-2018a-HDF5-1.10.1.eb

```diff
@@ -0,0 +1,19 @@
+name = 'netCDF-Fortran'
+version = '4.4.4'
+versionsuffix = '-HDF5-1.10.1'
```

Comment:
exactly the same as the existing `netCDF-Fortran-4.4.4-intel-2018a.eb`, no need for `versionsuffix`